pkg/sink/codec/common/helper.go

```diff
@@ -279,7 +280,7 @@ func queryRowChecksumAux(
 		}
 
 		log.Error("set snapshot read failed",
-			zap.String("query", query),
```

Comment:
Do we need to redact these query here?

Reply:
For all decoder.go in pkg/sink/codec, it's just a inner use for intergration tests or unit tests. I think we may don't need react values here?